sunpy/net/dataretriever/sources/fermi_gbm.py

```diff
@@ -1,15 +1,17 @@
 from sunpy.util.scraper import Scraper
 from ..client import GenericClient
 
+
 __all__ = ['GBMClient']
 
 
 class GBMClient(GenericClient):
     """
     Provides access to data from the Gamma-Ray Burst Monitor (GBM) instrument
```

Comment:
So clients which break the one line short description at the top of the docstring rule don't render so well in the `Fido` table. The indentation whitespace is there. I guess we could use dedent or just reword these docstrings a little?

```
        GBMClient Provides access to data from the Gamma-Ray Burst Monitor (GBM) instrument    on board the Fermi satellite.
```

Reply:
Fixed.